### DISCUSSION TITLE
labels of spoofing scores in the implemention of "One-class Learning Towards Synthetic Voice Spoofing Detection"

### DISCUSSION THREAD

**Discussion by avishai111** [yzyouzhang/AIR-ASVspoof#46] [Q&A] [answered]
In the function of t-dcf that ASV2019 published, the labels for bonafide or spoofing must be 1 and 0 accordingly no? 
![image](https://github.com/yzyouzhang/AIR-ASVspoof/assets/62557714/fa68ec88-5a3b-4636-bd41-aba1ed2d1061)

**Comment by yzyouzhang** [comment, marked as answer]
Referring to https://github.com/yzyouzhang/AIR-ASVspoof/blob/master/eval_metrics.py#L157, the target is bonafide speech, and the non-target is spoofing attacks. I use bonafide 0 and spoofing 1 in my code, but reversing is also fine, as long as you are consistent.
